Makefile

```diff
@@ -15,7 +15,10 @@ install: ## Install the dependencies
 
 webpage: ## Run the webpage localy
 	cd website && npm run start
-
+	
+send_mail: ## Send newsletter mail
+	cd curator && npx ts-node ./src/mail_agent/newsletter_script.ts
```

Comment:
**suggestion:** You should use workspace command here instead of changing directory